tests/app/main/views/test_stats.py

```diff
@@ -23,6 +23,18 @@ def test_get_stats_page(self, data_api_client):
 
         assert response.status_code == 200
 
+    def test_get_stats_page_for_open_framework_includes_framework_stats(self, data_api_client):
+        data_api_client.find_audit_events.return_value = {
+            'auditEvents': []
+        }
+        data_api_client.get_framework.return_value = {
+            'frameworks': {'status': 'open', 'lots': []}
+        }
+        response = self.client.get('/admin/statistics/g-cloud-7')
+
+        assert data_api_client.get_framework_stats.call_args == mock.call('g-cloud-7')
```

Comment:
This can use `called_once_with`

Reply:
I'm usually reluctant to use `assert_called_once_with` for 1) fear of a stray typo on a mock method always returning `True` 2) the clarity of having an `assert` at the beginning of the statement.   If we want to retain the ability ensure the mock is only called once we can use `call_args_list == [foo]` (probably overkill here). However I agree that whatever we use it should be consistent, which it clearly isn't below 😸 What do you think?